cli: cockroach workload run movr causes a panic

This is based on `master`:

```
cockroach start \
--insecure \
--store=node1 \
--listen-addr=localhost:26257 \
--http-addr=localhost:8080 \
--join=localhost:26257,localhost:26258,localhost:26259 \
--background
```

```
cockroach start \
--insecure \
--store=node2 \
--listen-addr=localhost:26258 \
--http-addr=localhost:8081 \
--join=localhost:26257,localhost:26258,localhost:26259 \
--background
```

```
cockroach start \
--insecure \
--store=node3 \
--listen-addr=localhost:26259 \
--http-addr=localhost:8082 \
--join=localhost:26257,localhost:26258,localhost:26259 \
--background
```

```
cockroach workload init movr 'postgresql://root@localhost:26257?sslmode=disable'
```
```
I190917 20:38:38.061574 1 workload/workloadsql/dataload.go:135  imported users (0s, 50 rows)
I190917 20:38:38.073601 1 workload/workloadsql/dataload.go:135  imported vehicles (0s, 15 rows)
I190917 20:38:38.148419 1 workload/workloadsql/dataload.go:135  imported rides (0s, 500 rows)
I190917 20:38:38.198438 1 workload/workloadsql/dataload.go:135  imported vehicle_location_histories (0s, 1000 rows)
I190917 20:38:38.264506 1 workload/workloadsql/dataload.go:135  imported promo_codes (0s, 1000 rows)
```
```
cockroach workload run movr --duration=5m 'postgresql://root@localhost:26257?sslmode=disable'
```
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x6976932]

goroutine 8 [running]:
github.com/cockroachdb/cockroach/pkg/workload/faker.(*weightedEntries).Rand(0x0, 0xc000366080, 0x24, 0xc000c083c0)
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/faker/faker.go:58 +0x72
github.com/cockroachdb/cockroach/pkg/workload/faker.(*addressFaker).StreetAddress(0xc000524a90, 0xc000366080, 0x24, 0x24)
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/faker/address.go:29 +0x3c
github.com/cockroachdb/cockroach/pkg/workload/movr.(*movr).Ops.func4(0x843a7e0, 0xc000318140, 0x0, 0x0)
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/movr/movr.go:692 +0x969
github.com/cockroachdb/cockroach/pkg/workload/cli.workerRun(0x843a7e0, 0xc000318140, 0xc000bcc0c0, 0xc00030c030, 0x0, 0xc0002d4000)
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/cli/run.go:235 +0x94
github.com/cockroachdb/cockroach/pkg/workload/cli.runRun.func1.1(0xc0003fd160, 0x7ac69d5, 0x4, 0xc0000ec020, 0x1, 0x1, 0x0, 0x0, 0x0, 0xc000bcc0c0, ...)
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/cli/run.go:401 +0xfd
created by github.com/cockroachdb/cockroach/pkg/workload/cli.runRun.func1
	/Users/jesseseldess/go/src/github.com/cockroachdb/cockroach/pkg/workload/cli/run.go:391 +0x133
```

--- a/pkg/workload/movr/movr.go
+++ b/pkg/workload/movr/movr.go
@@ -526,6 +526,10 @@ type rideInfo struct {
 
 // Ops implements the Opser interface
 func (g *movr) Ops(urls []string, reg *histogram.Registry) (workload.QueryLoad, error) {
+	// Initialize the faker in case it hasn't been setup already.
+	g.fakerOnce.Do(func() {
+		g.faker = faker.NewFaker()
+	})
 	sqlDatabase, err := workload.SanitizeUrls(g, g.connFlags.DBOverride, urls)
 	if err != nil {
 		return workload.QueryLoad{}, err

